lib/capability/capability.go

```diff
@@ -37,13 +37,13 @@ func (caps capabilitiesSort) Less(i, j int) bool { return string(caps[i]) < stri
 // SetCapabilities populates and returns cluster capabilities from ClusterVersion capabilities spec. This method also
 // ensures that no previously enabled capability is now disabled and returns any such implicitly enabled capabilities.
 func SetCapabilities(config *configv1.ClusterVersion,
-	existingEnabled, alwaysEnabled map[configv1.ClusterVersionCapability]struct{}) ClusterCapabilities {
+	existingEnabled []configv1.ClusterVersionCapability) ClusterCapabilities {
 
 	var capabilities ClusterCapabilities
-	capabilities.KnownCapabilities = setKnownCapabilities()
+	capabilities.KnownCapabilities = GetCapabilitiesAsMap(configv1.KnownClusterVersionCapabilities)
 
 	capabilities.EnabledCapabilities, capabilities.ImplicitlyEnabledCapabilities = setEnabledCapabilities(config.Spec.Capabilities,
-		existingEnabled, alwaysEnabled)
+		existingEnabled)
```

Comment:
> The argsexistingEnabled and alwaysEnabled were handled in the same way in the function SetCapabilities. We removed the alwaysEnabled in the function and send their union when calling the function.

The refactoring is structurally sound, but I think we need to accompany it with some renames. Previously this method accepted `existingEnabled` and `alwaysEnabled`. Now it accepts their _union_, so I think the parameter should no longer be called `existingEnabled`. I'm not sure about what name describes the union well though. The GoDoc should also describe what the parameter is.